File: server/src/main/java/com/testknight/TestBuddyTelemetryServer/exceptions/InvalidActionIdException.java
package com.testknight.TestBuddyTelemetryServer.exceptions;

public class InvalidActionIdException extends ValidationException {

    public static final long serialVersionUID = 927626852988215140L;


    public InvalidActionIdException(String actionId) {
        super(actionId + " is not a valid actionId");
    }

}

File: plugin/src/main/kotlin/com/testknight/exceptions/InvalidActionIdException.kt
package com.testknight.exceptions

import com.intellij.notification.NotificationType

class InvalidActionIdException(actionId: String) : TestKnightException() {
    override var title: String = "Invalid Action ID"
    override var content: String = "$actionId is not a valid action ID"
    override var type: NotificationType = NotificationType.ERROR
}
